vimeo-networking/src/main/java/com/vimeo/networking/model/error/ErrorCode.java

```diff
@@ -217,6 +217,8 @@
     URL_UNAVAILABLE,
     @SerializedName("2406")
     USER_NOT_AUTHORIZED_TO_DELETE_ACCOUNT,
+    @SerializedName("3102")
+    INVALID_INPUT_PASSWORD_TOO_LONG,
```

Comment:
Based on the comments/organization in this file it looks like this might fit better around line [156](https://github.com/vimeo/vimeo-networking-java/blob/f92daa0a47efc50d4d7ddc5c2b5af5301facd92c/vimeo-networking/src/main/java/com/vimeo/networking/model/error/ErrorCode.java#L156). That section looks like it's Auth->Input which this seems to fall into.